packages/logger/src/generators/winston/generator.spec.ts

```diff
@@ -26,29 +46,95 @@ describe('logger generator', () => {
         await generator(tree, {
             ...options,
             directory: 'custom',
-            tags: 'test, client',
+            tags: 'test, logger',
         });
 
-        const config = readProjectConfiguration(tree, 'custom-test-client');
+        const config = readProjectConfiguration(tree, 'custom-test-logger');
 
         expect(config).toBeDefined();
-        expect(config.tags).toEqual(['test', 'client']);
-        expect(tree.exists('test-client/src/index.ts'));
-        expect(tree.exists('test-client/src/index.test.ts'));
+        expect(config.tags).toEqual(['test', 'logger']);
+        expect(tree.exists(`${appName}/src/index.ts`));
+        expect(tree.exists(`${appName}/src/index.test.ts`));
+
+        expect(() =>
+            checkFilesExistInTree(
+                tree,
+                `custom/${appName}/tsconfig.json`,
+                `custom/${appName}/README.md`,
+                `custom/${appName}/tsconfig.lib.json`,
+                `custom/${appName}/tsconfig.spec.json`,
+                `custom/${appName}/package.json`,
+                `custom/${appName}/project.json`,
+                `custom/${appName}/.eslintrc.json`,
+                `custom/${appName}/jest.config.ts`,
+                `jest.config.ts`,
+                `jest.preset.js`,
+                `.prettierignore`,
+                `.prettierrc`,
```

Comment:
There is some duplication here with the snapshotFiles check below.
Where we check a file in snapshotFiles (e.g project.json, tsconfig.json) we do not need to verify them within this check aswell.

The snapshotFiles function by default already calls chechFilesExistInTree on all files which you pass to it

Reply:
Removed the individual checkFilesExistInTree check